modules/generators/content/password_file/secgen_local/local.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/ruby
+require_relative '../../../../../lib/objects/local_string_generator.rb'
+require 'erb'
+require 'fileutils'
+class PasswordFileGenerator < StringGenerator
+  attr_accessor :website_sample
+  attr_accessor :username_sample
+  attr_accessor :pass_list
+  LOCAL_DIR = File.expand_path('../../',__FILE__)
+  INTERESTS_DIR = "../../../../../lib/resources/interests"
+  TEMPLATE_PATH = "#{LOCAL_DIR}/templates/password_file.md.erb"
+
+  def initialize
+    super
+    self.pass_list = Array.new
+  end
+  
+  def get_options_array
+    super + [['--passwords', GetoptLong::REQUIRED_ARGUMENT]]
+  end
+
+  def process_options(opt, arg)
+    super
+    case opt
+    when '--passwords'
+      arr = arg.split(',', -1)
+      arr.each{ |pass|
+        self.pass_list << pass.strip!
+      }
+    end
+  end
+
+def generate
+
+  random_interest = Dir.glob(File.join("#{INTERESTS_DIR}/benign/", '*')).select { |f| File.directory? f }.sample
+
+  malicious_interest = Dir.glob(File.join("#{INTERESTS_DIR}/malicious/", '*')).select { |f| File.directory? f}.sample
+
+  website_lines = File.readlines("#{random_interest}/websites").map(&:strip)
+  
+  mal_website_lines = File.readlines("#{malicious_interest}/websites").map(&:strip)
```

Comment:
Ideally the benign and malicious interests would be fed in via parameters, but this is a good start, as is, so maybe ok to merge. Would be nice to have though.